website/project/metadata/global-flourishing-preregistration.json

```diff
@@ -0,0 +1,330 @@
+{
+   "atomicSchema":true,
+   "name":"Global Flourishing Study Descriptive Meta-analysis Core Team Wave One Preregistration",
+   "version":1,
+   "description":null,
+   "blocks":[
+      {
+         "block_type":"question-label",
+         "display_text":"Description",
+         "help_text":"Please give a brief description of your study, including background, purpose of the study, and your specific research question(s).",
+         "required":true,
```

Comment:
required field should be set on the input block, not the question-label block. Looks like this was an error in the sheet, but needs to be fixed (note that this seems to be wrong on all of the questions in the Global Flourishing schemas).